fvm/tests/default_kernel/dummy.rs

```diff
@@ -0,0 +1,250 @@
+use std::borrow::Borrow;
```

Comment:
Can we move this into the root "dummy" module? It seems like it may be useful in other tests?